pkgs/development/python-modules/deltalake/default.nix

```diff
@@ -64,6 +65,7 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     pytestCheckHook
     pandas
+    polars
```

Comment:
This should be in `optional-dependencies.polars` instead: https://github.com/delta-io/delta-rs/blob/python-v0.25.5/python/pyproject.toml#L41
Please also add the other optional dependencies.

Reply:
Are optional-dependencies available in the check phase? Because polars is needed for pytest to succeed.